ziggurat-core-crawler/src/summary.rs

```diff
@@ -0,0 +1,70 @@
+use std::{cmp, collections::HashMap, fmt, fs, path::Path, time::Duration};
+
+use serde::Serialize;
+use spectre::graph::AGraph;
+
+#[derive(Default, Clone, Serialize)]
+pub struct NetworkSummary {
+    pub num_known_nodes: usize,
+    pub num_good_nodes: usize,
+    pub num_known_connections: usize,
+    pub num_versions: usize,
```

Comment:
If there are some fields that could be not clear (like that one you've talked about) it'd be good to have some documentation here what's field meaning.

Reply:
Added some comments.